lib/amazon2.js

```diff
@@ -191,7 +191,7 @@ function send(path, errorMap, cb) {
 			res.service = constants.SERVICES.AMAZON;
 		}
 		catch (exc) {
-			exc.message += ': ' + path;
+			exc.message += ': ' + path + ", statusCode: " + response.statusCode;
```

Comment:
Would you please change the double quotes to single quotes to match the rest of the code?
Otherwise it looks greate!